thor/orbits/iod.py

```diff
@@ -180,7 +180,45 @@ def iod_worker(
     return iod_orbits, iod_orbit_members
 
 
-iod_worker_remote = ray.remote(iod_worker)
+@ray.remote
+def iod_worker_remote(
+    linkage_ids: Union[npt.NDArray[np.str_], ray.ObjectRef],
+    linkage_members_indices: Tuple[int, int],
```

Comment:
We discussed this previously but I wanted to reiterate that the pattern of passing indices to the ray remotes and letting that function handle the slicing is a good design choice. It's a clear delineation between ray vs processing in serial.